app/email_utils.py

```diff
@@ -182,10 +182,12 @@ def send_invalid_totp_login_email(user, totp_type):
         "Unsuccessful attempt to login to your SimpleLogin account",
         render(
             "transactional/invalid-totp-login.txt",
+            name=user.name,
```

Comment:
as `user.name` can be None, I think we can use `name = user.name or ""` here to make sure the email never displays `None`

Reply:
Done, I think it's better to change it in the email template though so we don't end up with any formatting issues.